Sources/URLEncodedForm/Data/URLEncodedFormSerializer.swift

```diff
@@ -3,19 +3,35 @@ import Bits
 /// Converts `[String: URLEncodedFormData]` structs to `Data`.
 final class URLEncodedFormSerializer {
     /// Default form url encoded serializer.
-    static let `default` = URLEncodedFormSerializer()
+    static let `default` = URLEncodedFormSerializer(sortedKeys: false)
+    
+    /// Produce JSON with dictionary keys sorted in lexicographic order.
+    private var sortedKeys: Bool
 
     /// Create a new form-urlencoded data serializer.
-    init() {}
+    init(sortedKeys: Bool) {
+        self.sortedKeys = sortedKeys
+    }
 
     /// Serializes the data.
     func serialize(_ URLEncodedFormEncoded: [String: URLEncodedFormData]) throws -> Data {
         var data: [Data] = []
-        for (key, val) in URLEncodedFormEncoded {
+        var elements: [(key: String, value: URLEncodedFormData)]!
```

Comment:
We should be able to avoid using an IUO here since the compiler can be sure we set `elements` w/ the following if / else.